presto-main/src/main/java/com/facebook/presto/operator/SortedPositionLinks.java

```diff
@@ -43,7 +42,8 @@
  *
  * filterFunction_1(...) OR filterFunction_2(....) OR ... OR filterFunction_n(...)
  *
- * To use lessThanFunction in this class, it must be an expression in form of:
+ * To use the elements in the list of filters inequalityJoinFilterConjuncts in this class,
```

Comment:
Yeah - but in this case it is essentially a symbol when it comes to filter function in join.
I would rather keep in javadoc that g(b_symbol) is not supported. As this javadoc is describing what is supported locally.
It should not need to be updated anytime we add some optimizer in other place which transforms expression which does not have supported shape to one which has.

Then this Javadoc would be unmaintainble (if it is not such already :) )